cypress/integration/expanded_spec.js

```diff
@@ -1,93 +1,117 @@
 beforeEach(() => {
-  cy.visit('http://localhost:3000/dashboard')
+  cy.visit('http://localhost:3000/movies')
```

Comment:
As discussed - we will probably change this to just be backslash as reflected in the spec and to help with error handling bad urls.